db/people.yml

```diff
@@ -464,7 +464,12 @@ cbullard:
   active: true
   aka: []
   avatar: https://github.com/chloeann95.png
-  bio: Chloë grew up in Michigan and attended Michigan State University for her undergraduate degrees in Physics and Mechanical Engineering, where she worked on computational plasma physics research. Afterward she attended the University of Toledo completing a Masters in Chemical Engineering while working on modeling CFD simulations for biomass with NREL. When not working you can find her looking for a new game to play or a new song to learn on the piano.$
+  bio: Chloë grew up in Michigan and attended Michigan State University for her undergraduate
+       degrees in Physics and Mechanical Engineering, where she worked on computational plasma
+       physics research. Afterward she attended the University of Toledo completing a Masters
+       in Chemical Engineering while working on modeling CFD simulations for biomass with NREL.
+       When not working you can find her looking for a new game to play or a new song to learn 
+       on the piano.$
```

Comment:
there seems to be a trailing dollar sign?  

I think this indentation will probably work but we usually only indent 2 spaces in our yml, so the first letter of subsequent lines would line up with the "o" of bio.  If you roundtrip the file, it may do it for you actually.  This will happen if you try and build anything using regolith, e.g., running `regolith build cv --people cbullard` from rg-db-group/local.   There is also a roundtrip.py file in that same folder so you don't have to build anything.

Reply:
I think the $ sign was from being copied from the HTML file I used so I didn't need to rewrite it. I'll fix it and the indentation. I wasn't sure how to line up the intending so I lined it up with the previous lines for the bio.